src/java.desktop/unix/native/common/awt/X11Color.c

```diff
@@ -1233,11 +1231,10 @@ jobject awtJNI_GetColorModel(JNIEnv *env, AwtGraphicsConfigDataPtr aData)
 void
 awt_allocate_systemrgbcolors (jint *rgbColors, int num_colors,
                               AwtGraphicsConfigDataPtr awtData) {
-    int i, pixel;
-    for (i = 0; i < num_colors; i++)
-        pixel = alloc_col (awt_display, awtData->awt_cmap, red (rgbColors [i]),
-                           green (rgbColors [i]), blue (rgbColors [i]), -1,
-                           awtData);
+    for (int i = 0; i < num_colors; i++)
```

Comment:
This requires C99. Is this ok for all platforms with all supported compilers?

Reply:
Yes, we have C11 as minimum for the JDK.